SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.16.x  | :white_check_mark: |
+| < 0.16  | :x:                |
+
+## Reporting a Vulnerability
+
+If you believe you have found a security vulnerability, please DO NOT disclose it publicly until we’ve had a chance to fix it.
```

Comment:
We could add an issue template that points folks to the correct link.

Reply:
added!